src/services/Content/ContentQueries.ts

```diff
@@ -3,20 +3,24 @@ import { QueryClient, useMutation, useQuery } from 'react-query';
 
 import { useNotification } from './../Notifications/Notifications';
 import {
-  filterData,
   ContentListResponse,
-  ContentRepoParamsResponse,
   deleteContentListItem,
   getContentList,
-  getContentRepoParams,
+  RepositoryParamsResponse,
+  getRepositoryParams,
+  AddContentListItems,
+  CreateContentRequest,
+  FilterData,
+  validateContentListItems,
 } from './ContentApi';
 
 export const CONTENT_LIST_KEY = 'CONTENT_LIST_KEY';
-export const CONTENT_REPO_PARAMS_KEY = 'CONTENT_REPO_PARAMS_KEY';
+export const REPOSITORY_PARAMS_KEY = 'REPOSITORY_PARAMS_KEY';
+export const CREATE_PARAMS_KEY = 'CREATE_PARAMS_KEY';
 
-export const useContentListQuery = (page: number, limit: number, filterData: filterData) =>
+export const useContentListQuery = (page: number, limit: number, filterData: FilterData) =>
   useQuery<ContentListResponse>(
-    [CONTENT_LIST_KEY, page, limit, filterData],
+    [CONTENT_LIST_KEY, page, limit, ...Object.values(filterData)],
```

Comment:
@rverdile In order to track these values I needed to flatten out the filterData object to get the key to work. As with the current filterData, I think it was being detected as when the comparison is run, I was seeing the key print as Object[Object].